SAML/saml_config.sh

```diff
@@ -37,6 +38,144 @@ function print_usage() {
   exit 1
 }
 
+function get_settings_id() {
```

Comment:
We should check if we can avoid this code duplication between openid and saml and have a common class/file that gets imported.

Reply:
Yes, you are right. I wanted this scripts to be working as soon as possible. And my plan was to probably refactor all of them when updating LDAP script (including sync features). Can we live by this duplication by now?

Reply:
Now that I'm implementing the changes I believe we should keep this without this refactor:
* We can extract around 30-40 lines of code (which is more than OK)
* If we extract that code we should make it live in a separate folder:
** Then if someone wants to send these scripts using a tarball it will be more likely to fail

I agree in refactoring this kind of code. But probably because of this we should start adding `admin` endpoints to `sdclient` or maybe starting a `sdc_admin_client` so we can properly deliver it to customers (it might be even useful for us to avoid having this code replicated in more than one place).

Reply:
The refactor is not a blocker, but should be addressed soon.